### ISSUE TITLE
Can't recognize install path on Mac

### ISSUE THREAD

**Issue by NattyNarwhal** [Lymia/MPPatch#80]
I'm trying to install the patch on macOS. If I open MPPatch, I get `MPPatch cannot find the Civilization V installation folder.`.

If I try manually specifying a path in settings, I get `Civilization V does not appear to be installed in this folder.`. Doesn't matter where; I've tried:

* `/Users/calvin/Library/Application Support/Steam/steamapps/common/Sid Meier's Civilization V/Civilization V.app/Contents/MacOS`
* `/Users/calvin/Library/Application Support/Steam/steamapps/common/Sid Meier's Civilization V/Civilization V.app`
* `/Users/calvin/Library/Application Support/Steam/steamapps/common/Sid Meier's Civilization V`
* `/Users/calvin/Library/Application Support/Steam/steamapps/common`
* `/Users/calvin/Library/Application Support/Steam/steamapps`

Steam is using the default library folder:

```
/Users/calvin/Library/Application Support/Steam/steamapps/common/Sid Meier's Civilization V/Civilization V.app
```

Validating doesn't help, as it's installed and I can launch it just fine.

I'm using the Azul JDK:

```
openjdk version "17.0.6" 2023-01-17 LTS
OpenJDK Runtime Environment Zulu17.40+19-CA (build 17.0.6+10-LTS)
OpenJDK 64-Bit Server VM Zulu17.40+19-CA (build 17.0.6+10-LTS, mixed mode, sharing)
```

This is macOS 13.4 on an M1 Pro MBP.

**Comment by NattyNarwhal** [comment]
![image](https://github.com/Lymia/MPPatch/assets/3161292/70a888cc-e8d0-40e1-a82e-3a563d8bfdce)
uhhhh I don't think it's normal for a JVM to look up these paths

(FWIW, `lMachines` seems to come from the JVM path at `/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home`)

**Comment by NattyNarwhal** [comment]
![Screenshot 2023-06-11 at 12 21 35 AM](https://github.com/Lymia/MPPatch/assets/3161292/3b0974ec-8991-4a23-b732-2de115c4201e)

According to `MacOSPlatform` I should be using `/Users/calvin/Library/Application Support/Steam/steamapps/common/Sid Meier's Civilization V`. If I manually select that, it seems to choke still...

I've tried OpenJDK 11 and OpenJDK 8 (which is x86 only, so Java is running under emulation) instead of Azul's 17, and no difference.

**Comment by NattyNarwhal** [comment]
![Screenshot 2023-06-11 at 12 31 56 AM](https://github.com/Lymia/MPPatch/assets/3161292/1ca660cf-a754-4574-9f1e-12de3ac800a0)
I think that's a red herring regardless though - Vsub is actually real according to https://github.com/Lymia/MPPatch/blob/8adb8b735e884f26ae8e750339bbf213f8b556a4/src/patch/install/macos.xml#L30-L31, but it also doesn't exist in this current build of the game?

**Comment by Lymia** [comment]
Yeah, the macOS version is extremely unsupported since I have very spotty access to a development machine for it. I used to have a hackintosh desktop when I lived with my parents, but I no longer have access to that computer. Going to be a mess to try and set another one up (since I've been using AMD computers lately), but I might see if I can get that to work sometime later.

In the meanwhile, I'm going to release 0.2.0 with no native macOS support, and rely instead on the Proton version, since that shouldn't be too different from Proton-on-Linux.

**Comment by NattyNarwhal** [comment]
There's no Proton on macOS though?

**Comment by Lymia** [comment]
Urgh. I'll see what I can do.

**Comment by B86349** [comment]
Is there a no way of putting the instal path in manually?
